lib/dashed/button.rb

```diff
@@ -37,7 +37,7 @@ def capture
     end
 
     def duplicate_arp?
-      last_press && (last_press - Time.now).abs < 45
+      last_press && (last_press - Time.now).abs < @timeout
```

Comment:
Good catch. What are your thoughts on just using `timeout` here instead of the ivar? Also, looking at it further, there is really no reason why it should be an `attr_accessor`, I think we would be fine using `attr_reader`.